UnrealPlugin/README.md

```diff
@@ -46,5 +124,53 @@ public class <projectname>ServerTarget : TargetRules
 		DisablePlugins.Add("MediaFoundationMediaPlayer");
 	}
 }
+```
+
+There are two ways to include the app-local prerequisites - either through the Unreal Engine editor or by editing DefaultGame.ini.
+###### In the Unreal Editor
+In the editor go to Edit -> Project Settings. In the opened window navigate to Packaging on the left side. Scroll to the bottom of the list, and tick "Include app-local prerequisities".
+
+###### In DefaultGame.ini
+Or you can update DefaultGame.ini to show the following:
+ [/Script/UnrealEd.ProjectPackagingSettings]
+IncludeAppLocalPrerequesites=True
+
+If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
+
+If you are using Continuous Integration (CI), then you could add it to your setup to only turn this flag on when building a dedicated server, so the additional dlls only get added if it is a dedicated server build.
+
+Update [game name].Build.cs file to add "PlayfabGSDK" into the PublicDependencyModuleNames.AddRange(); list as follows:
+
+```csharp
+		PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "HeadMountedDisplay", "PlayfabGSDK"});
+
+		PrivateDependencyModuleNames.AddRange(new string[] { });
+```
+
+There are two ways to update the Game Instance Class - either through the Unreal Engine editor or by editing DefaultEngine.ini directly. 
+###### In the Unreal Editor
+In the editor, This can also be set through the UI in the editor. In the editor go to Edit -> Project Settings. From that opened window,
+navigate to Maps&Modes on the left side. Scroll to the bottom, and then you can set the option "Game Instance Class" to your new game instance class directly, and avoid typos.
+
+###### In DefaultEngine.ini
+Or you can update DefaultEngine.ini file and add this:
+[/Script/EngineSettings.GameMapsSettings]
+GameInstanceClass=/Script/[game name].MyGameInstance
+
+Then select the option to build from development editor mode and build the project.
+
+![image depicting Visual Studio with the option to build in Development Editor Mode](Documentation/DevelopmentEditor.png)
+
+Then select the option to build from development server mode and build the project again.
+
+Once the build succeeds, close Visual Studio.
+
+## Packaging the game server
+
+Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files". 
+
+Then open the .uproject file.
 
-```
\ No newline at end of file
+In the top left of the Unreal editor menu, click on file->package->target configuration ->[game name] server and then go to file -> package -> windows x64. 
```

Comment:
File->Package->Target Configuration ->[game name]Server

and File -> Package -> Windowsx64